content/blog/bitigchi/2020-10-01-haiku_monthly_activity_report_september_2020.md

```diff
@@ -0,0 +1,96 @@
+
+AlwaysLivid added Mail auto-configure data for mailbox.org and riseup.net.
+
+bitigchi fixed the redundant reboot confirmation in SoftwareUpdater.
+
+Pascal Abresch made TeamMonitor and AboutSystem compliant to the system colours; previously the text colours would not react properly to dark system themes.
+
+PulkoMandy fixed various issues with MediaPlayer and ffmpeg implementation that caused seeking and cover art display problems.
+
+Murai Takashi made some code cleanups in various system parts.
+
+syedsouban and bitigchi implemented custom search engine shortcuts in WebPositive. Currently following shortcuts are supported:
+
+- Google (g)
+- Bing (b)
+- Wikipedia (w)
+- DuckDuckGo (d)
+- Baidu (a)
+- Yandex (y)
+- Ecosia (e)
+- Qwant (q)
+
+## Compatibility
+
+CodeForEvolution implemented the `qsort_r` function.
+
+## Documentation
+
+John Scipione improved the documentation for BMessage and AppKit.
+
+nielx implemented layout customisations for the Doxygen stylesheet.
+
+## File Systems
+
+brjhaiku implemented the initial methods required for btrfs write support.
+
+## Kernel
+
+Jerome Duval implemented support for more CPU power modes and functions that allows system to use the CPU power more efficiently. This fixed an issue where on some hardware the system clock speed was stuck at 1.5 GHz, and improved performance on other systems.
+
+## Package Management
+
+Andrew Lindesay fixed a crash in HaikuDepot whilst synchronising package metadata.
+
+## Preferences
+
+Preetpal Kaur's GSOC project work has been merged, and she continues to work on final polishes and bug fixes.
+
+## POSIX
+
+rofl0r has fixed the prototype of gethostbyaddr to be more in-line with FreeBSD and Linux.
+
+## RISC-V
+
+kallisti5 added u-boot support for RISC-V.
+
+## System Kits
+
+John Scipione fixed a regression that prevented character drawing when a tab is inserted.
+
+Pascal Abresch implemented enabling adding custom user TLS certificates.
+
+PulkoMandy implemented creating catalog files from external source files.
+
+## System Servers
+
+PulkoMandy fixed a regression that caused unpersistant brightness levels across different workspaces. This eventually led fixing a system crash as well.
+
+## System Translators
+
+Gerasim Troeglazov fixed a crash that occurs while displaying `.icns` files.
+
+------
+
+If you would like to try out these changes right away, grab a copy of the latest [nightlies](https://download.haiku-os.org) and give it a go!
+
+Haiku Project always welcomes new faces. If you are a developer, sysadmin, designer, or a writer, we will always appreciate your support. Come meet us at the [Freenode #haiku IRC channel](irc://chat.freenode.net/haiku),  [forums](https://discuss.haiku-os.org) or introduce yourself in the [Haiku mailing list](https://www.freelists.org/list/haiku).
```

Comment:
I'd say "new hands" rather than "new faces"?

Reply:
It's a nice pun, but sounds too meta. As if we only welcome the work done by hands(?!) rather than the person him/herself.

Reply:
well that's how it would be in French, I think. But maybe it doesn't work too well in English then.